controlpanel/frontend/jinja2/help.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% set page_title = "Help" %}
+
+{% block content %}
+  <div class="govuk-grid-row" id="reset-container">
```

Comment:
I'm assuming this `id="reset-container"` is just leftover from some other view ☺️ (I don't think it should matter anyway unless we have any JS that uses these IDs, right?)

Reply:
Doh. Correct. Will fix. ;-)